core/src/main/java/com/google/instrumentation/stats/StatsContextFactory.java

```diff
@@ -35,4 +40,69 @@
    * Returns the default {@link StatsContext}.
    */
   public abstract StatsContext getDefault();
+
+  /**
+   * Get current StatsContext from current gRPC Context.
+   *
+   * @return the current {@code StatsContext} from {@code io.grpc.Context}, or {@code null} if
+   *     there's no {@code StatsContext} associated with current {@code io.grpc.Context}.
+   */
+  @Nullable
+  public final StatsContext getCurrentStatsContext() {
+    // TODO(songya): return a no-op StatsContext (or default) when getCurrentStatsContext() returns
```

Comment:
I think we should do something here because returning null will make a lot of users to write:

StatsContext current = ...
if (current != null) {
  ...
}

So probably we should return the default. @dinooliva do you have any preference here?